src/Microsoft.ML.AutoML/Sweepers/SweeperProbabilityUtils.cs

```diff
@@ -98,13 +99,15 @@ public static float[] ParameterSetAsFloatArray(IValueGenerator[] sweepParams, Pa
                 }
                 else if (sweepParam is LongValueGenerator lvg)
                 {
+                    var longValue = GetIfIParameterValueOfT<long>(pset) ?? long.Parse(pset.ValueText);
```

Comment:
To resurrect an earlier comment by @antoniovs1029, using CultureInvariant for parsing a long would be safer. I don't know of any cultures that differ from the default but a user can set their machine to a custom culture.

The ToString portion would also need a matching CultureInvariant set: https://github.com/dotnet/machinelearning/blob/d849ba4c7831586018d5b2cca6bdd2b1fa228ddf/src/Microsoft.ML.AutoML/Sweepers/Parameters.cs#L86

Reply:
```suggestion
                    var longValue = GetIfIParameterValueOfT<long>(pset) ?? long.Parse(pset.ValueText, CultureInfo.InvariantCulture);
```

Reply:
@antoniovs1029: I see that @ptelman does allow changes into his branch. Alternatively, do you want to put the fix for the `long` in your other PR (https://github.com/dotnet/machinelearning/pull/5462)?

Reply:
@justinormont sorry for not being active. Please go ahead and push any changes which you think are required to have this merged.

Reply:
Thanks.
#Resolved